tests/scripts/run-xunit-perf.cmd

```diff
@@ -266,6 +287,7 @@ setlocal
   set LV_SUBMISSION_ARGS=%LV_SUBMISSION_ARGS% --config-name "%TEST_CONFIG%"
   set LV_SUBMISSION_ARGS=%LV_SUBMISSION_ARGS% --config Configuration "%TEST_CONFIG%"
   set LV_SUBMISSION_ARGS=%LV_SUBMISSION_ARGS% --config OS "Windows_NT"
+  set LV_SUBMISSION_ARGS=%LV_SUBMISSION_ARGS% --config ETW_Collection "%ETW_COLLECTION%"
```

Comment:
I would make this more generic: `--config Profile [ON|OFF]`, so it will make sense and match the Linux configurations.